plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -81,9 +81,10 @@ def _createDialog(self, qml_name):
     @pyqtSlot(result = str)
     def getExampleData(self) -> str:
         if self._example_data_content is None:
-            file_path = os.path.join(PluginRegistry.getInstance().getPluginPath(self.getPluginId()), "example_data.json")
-            with open(file_path, "r", encoding = "utf-8") as f:
-                self._example_data_content = f.read()
+            file_path = os.path.join(cast(str, PluginRegistry.getInstance().getPluginPath(self.getPluginId())), "example_data.json")
```

Comment:
If getPluginPath(...) returns None for some reason, the statement will fail.

Reply:
Fixed in ac0192f